apps/frontend/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -6,6 +6,7 @@ import { ICON_SIZE } from "./libs/constants/constants.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	className?: string;
```

Comment:
We shouldn't be able to styles common component like this. This is way it won't be COMMON component

Reply:
You can extend `variant` property with additonal variant, I guess

Reply:
Can't we use the already existing outlined variant here? Or adjust its styles slightly so that it fits both cases? It's better to keep as less variants as we can in the project ui overall and for common components too

Reply:
I talked to Veronika, and we decided that we'll just use outlined variant then. Thanks!